web-app/src/main/java/com/google/sps/servlets/ReportServlet.java

```diff
@@ -70,6 +78,9 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
             response.getWriter().println(exception);
           }
           break;
+        case "image":
+          reportEntity.setProperty(paramName, getUploadedFileUrl(request, "image"));
```

Comment:
If you are not using value here, maybe it doesn't needs to be inside the loop & switch statement